servo/servo.dockerfile

```diff
@@ -53,6 +53,11 @@ WORKDIR servo
 # Configure the IDEs to use Servo's source directory as workspace.
 ENV WORKSPACE /home/user/servo/
 
+# Work around a Servo build problem.
+RUN echo "\n# Work around https://github.com/servo/servo/issues/20712." >> /home/user/.bashrc \
+ && echo "HARFBUZZ_SYS_NO_PKG_CONFIG=1" >> /home/user/.bashrc
```

Comment:
This should be `export HAFBUZZ_SYS_NO_PKG_CONFIG=1`

The version without export won't be visible to sub-processes of the shell, like cargo.

Reply:
Fair enough, thanks for catching this!

Interestingly, we're not consistent with this throughout this repository, as shown by [searching for "PATH"](https://github.com/JanitorTechnology/dockerfiles/search?q=PATH).